### ISSUE TITLE
Status of the project

### ISSUE THREAD

**Issue by scinos** [FormidableLabs/enzyme-matchers#353]
Hi!

What is the status of this project? Is it still being maintained?

Last commit is from almost 1 year ago, and since then there has been important issues like Node 14 support (#349) or Jest 26 support (#351) with no comment from the maintainers.

**Comment by ljharb** [comment]
The project was recently moved into the enzyme org; I'll try to address those issues and cut a release in the next few weeks.

**Comment by Kikobeats** [comment]
looks like this project is dead ☹️

**Comment by scinos** [comment]
How can I help?

**Comment by Kikobeats** [comment]
@ljharb if you give us the right permissions, we can push the project forward!

There are some easy PR that can be merged and keep the project up to date.

**Comment by AlonMiz** [comment]
agree. we are stuck right now because of the out of dated js dom.
that isn't working with react 17.
also in order for something to work, we needed this package
@wojtekmaj/enzyme-adapter-react-17

**Comment by LexaDel** [comment]
@ljharb Hi, Could you update jest-environment-jsdom in jest-environment-enzyme to 27.2.2 version to support the new jest?

**Comment by ljharb** [comment]
@LexaDel no, because i have zero association to `jest-environment-enzyme`, and this project (as well as enzyme itself) has no relation to jest.

**Comment by ljharb** [comment]
@LexaDel it's on my list of stuff to do.

**Comment by BerkeleyTrue** [comment]
any updates on this?

**Comment by GoudekettingRM** [comment]
@ljharb Thanks for doing this! Can imagine how much work it would be. Do you have a timeline for react 17 and node support?

**Comment by ljharb** [comment]
I do not have a timeline for react 17 support; i don’t know what “node support” it’s lacking.

**Comment by GoudekettingRM** [comment]
> I do not have a timeline for react 17 support; i don’t know what “node support” it’s lacking.

Sorry, the node support I added by mistake 😅  That's all working as it should. 

Thanks for the quick reply (:
